ui/shared/components/graph/VariantTagTypeBar.jsx

```diff
@@ -5,9 +5,12 @@ import HorizontalStackedBar from '../graph/HorizontalStackedBar'
 import { EXCLUDED_TAG_NAME, REVIEW_TAG_NAME, NOTE_TAG_NAME } from '../../utils/constants'
 
 
-export const getVariantTagTypeCount = (vtt, familyGuids) => (
-  familyGuids ? familyGuids.reduce((count, familyGuid) => count + (vtt.numTagsPerFamily[familyGuid] || 0), 0) : vtt.numTags
-)
+export const getVariantTagTypeCount = (vtt, familyGuids) => {
```

Comment:
you could probably just switch this back to the original ternary statment (`familyGuids ? familyGuids.reduce((count, familyGuid) => count + (vtt.numTagsPerFamily[familyGuid] || 0), 0) : vtt.numTags`)